Xamarin.Forms.Core/Cells/Cell.cs

```diff
@@ -18,6 +19,24 @@ public abstract class Cell : Element, ICellController
 
 		bool _nextCallToForceUpdateSizeQueued;
 
+		EffectiveFlowDirection _effectiveFlowDirection = EffectiveFlowDirection.LeftToRight | EffectiveFlowDirection.Implicit;
+		EffectiveFlowDirection IFlowDirectionController.EffectiveFlowDirection
+		{
+			get { return _effectiveFlowDirection; }
+			set
+			{
+				if (value == _effectiveFlowDirection)
+					return;
+
+				_effectiveFlowDirection = value;
+
+				var ve = (Parent as VisualElement);
+				ve?.InvalidateMeasureInternal(InvalidationTrigger.Undefined);
+				OnPropertyChanged(VisualElement.FlowDirectionProperty.PropertyName);
+			}
+		}
+		IFlowDirectionController FlowController => this;
```

Comment:
Style Police here. You're missing a white line. Please pull over !

(well, fix it only if you have other things to push)

Reply:
👮